manticore/core/state.py

```diff
@@ -296,6 +296,20 @@ def concretize(self, symbolic, policy, maxcount=7):
                 )
         elif policy == "ONE":
             vals = [self._solver.get_value(self._constraints, symbolic)]
+        elif policy == "OPTI":
+            logger.info("Optimistic case when forking")
+            if self._solver.can_be_true(self._constraints, symbolic):
+                vals = (True,)
+            else:
+                #We assume the path constraint was feasible to begin with
+                vals = (False,)
+        elif policy == "PESI":
+            logger.info("Pesimistic case when forking")
+            if self._solver.can_be_false(self._constraints, symbolic):
+                vals = (False,)
+            else:
+                #We assume the path constraint was feasible to begin with
```

Comment:
The comments for both cases are the same. Copy & Paste gaffe?

Reply:
No it is actually what I wanted to write there. 
This is forking and concretizing `symbolic`. If possible, "PESI" ignores the `True` option and "OPTI" ignores the `False` option. 

Preconditions & assumptions 
* `symbolic` is a Bool. 
* self._constraints are feasible (symbolic has at least one solution)
* solutions/vals can be: true, false or true|false